test/disability-benefits/526EZ/config/veteranInformation.unit.spec.jsx

```diff
@@ -38,7 +38,7 @@ describe('526EZ veteran information', () => {
 
     form.find('form').simulate('submit');
 
-    expect(form.find('.usa-input-error-message').length).to.equal(6);
+    expect(form.find('.usa-input-error-message').length).to.equal(5);
```

Comment:
I'm actually not sure how this was working before. It currently throws validation messages on the following fields: First name, last name, VA file number, gender, date of birth. It doesn't throw errors on: Middle name, SSN.